packages/core/src/bar-chart.ts

```diff
@@ -106,7 +106,9 @@ export class BarChart extends BaseAxisChart {
 						.attr("fill", d => this.getFillScale()[d.datasetLabel](d.label))
 						.attr("stroke", d => this.options.accessibility ? this.colorScale[d.datasetLabel](d.label) : null)
 						.attr("stroke-width", Configuration.bars.default.strokeWidth)
-						.attr("stroke-opacity", d => this.options.accessibility ? 1 : 0);
+						.attr("stroke-opacity", d => this.options.accessibility ? 1 : 0)
+						.attr("aria-label", d => this.x1(d.datasetLabel))
```

Comment:
Would need to be move to `makeAccessible()`, although for bar it might have to stay. Needs investigating